src/shared_.toit

```diff
@@ -0,0 +1,136 @@
+// Copyright (C) 2023 Toitware ApS.
+// Use of this source code is governed by a Zero-Clause BSD license that can
+// be found in the tests/TESTS_LICENSE file.
+
+import .fs
+
+/**
+Cleans the path.
+
+The $path must already use the $separator as separators. Typically this is achieved
+  by calling $from-slash.
```

Comment:
Not your code, but please apply.
```suggestion
The $path must use the native $separator as separators (`\\` on Windows, `/` otherwise). Typically
  this is achieved by calling $from-slash.
```